ios/Podfile.lock

```diff
@@ -76,116 +86,156 @@ PODS:
     - nanopb/encode (= 0.3.901)
   - nanopb/decode (0.3.901)
   - nanopb/encode (0.3.901)
-  - React (0.59.10):
-    - React/Core (= 0.59.10)
-  - react-native-netinfo (3.2.1):
-    - React
-  - react-native-splash-screen (3.2.0):
-    - React
-  - React/Core (0.59.10):
-    - yoga (= 0.59.10.React)
-  - React/CxxBridge (0.59.10):
+  - React (0.60.6):
```

Comment:
just for me to understand it:
you upgrade to react-native 0.61.5 or something like this, right? but why it is 0.60.6 here.

Reply:
Good catch. I think you saved the day here. It seems that I forgot to run the `pod install` command when upgrading to the latest RN, so it probably means that the test build was running on 0.60 on iOS, and 0.61 on android? Just upgraded pod files in https://github.com/mapswipe/mapswipe/commit/f0a797a4763a2b0827cc64dd1f47a1101ec62a87 In any case, we should probably do a bit more testing on iOS before releasing, just to make sure nothing is broken.